components/invitation_modal/invitation_modal.scss

```diff
@@ -1,6 +1,9 @@
 .InvitationModal {
     .modal-dialog {
-        position: absolute;
+        @media screen and (min-width: 640px) {
+            position: absolute;
+        }
+
```

Comment:
nit: For a screen width between 768 and 640px the dialog still shown a bit reduced:

https://user-images.githubusercontent.com/79058848/188924087-9f3a40a9-ca55-43e8-876c-954d1a31bc06.mp4